common/frame_manager.go

```diff
@@ -238,9 +238,11 @@ func (m *FrameManager) frameNavigated(frameID cdp.FrameID, parentFrameID cdp.Fra
 		m.ID(), frameID, parentFrameID, documentID, name, url, initial)
 
 	if frame != nil {
+		m.framesMu.Unlock()
```

Comment:
TODO: Create a e2e test for this. Need to work out how to create a frame in a frame, maybe with `iframe`s?

Reply:
Should we add it in this same PR?